x-pack/legacy/plugins/uptime/server/lib/adapters/framework/adapter_types.ts

```diff
@@ -29,6 +29,13 @@ export interface UMFrameworkRouteOptions<
   config?: any;
 }
 
+export interface KibanaCore {
```

Comment:
I'd suggest you call this type `UptimeCoreSetup` to make it clear that this is uptime's shim type, not the final shape of what CoreSetup will look like in a NP plugin.